virtual/activation_cache.go

```diff
@@ -83,16 +84,22 @@ func (a *activationsCache) ensureActivation(
 	moduleID,
 	actorID string,
 
-	blacklistedServerID string,
+	extraReplicas uint64,
+	blacklistedServerIDs []string,
 ) ([]types.ActorReference, error) {
 	// Ensure we have a short timeout when communicating with registry.
 	ctx, cc := context.WithTimeout(ctx, defaultActivationCacheTimeout)
 	defer cc()
 
+	isServerIdBlacklisted := make(map[string]bool)
```

Comment:
actually `blacklistedServerIDsSet` might be the best name

Reply:
I believe that using the name `isServerIdBlacklisted` would enhance readability, particularly when it is employed within "if" conditions. On the other hand, opting for `blacklistedServerIDsSet` would improve readability when it is included as part of a request, such as when it is assigned to a struct field. So, in this case, I would use `isServerIdBlacklisted`